website/components/footer.html

```
@@ -1,3 +1,6 @@
+<head>
+  <link rel="stylesheet" href="scripts/components/global.css">
```

HTMLHint (1.9.1)

[warning] The empty tag : [ link ] must be self closed.

(tag-self-close)